app/utils.py

```diff
@@ -68,7 +68,7 @@ def format_resource_search(hit):
         'url': hit['url'],
         'category': hit['category'],
         'languages': hit['languages'],
-        'paid': hit['paid'],
+        'free': hit['free'],
```

Comment:
oh snap, I forgot, this change is going to require us to do something about Algolia... I wonder if re-indexing will be enough? Anyway, nothing for you to do here, just wanted to make a note for myself that once we merge this and deploy we'll have to fix that

Reply:
I totally called it, seeing this in production
```
[2020-10-21 15:34:32,534] ERROR in app: Exception on /api/v1/search/ [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./app/api/routes/search.py", line 14, in search
    return search_results()
  File "./app/api/routes/search.py", line 72, in search_results
    results = [utils.format_resource_search(result) for result in search_result['hits']]
  File "./app/api/routes/search.py", line 72, in <listcomp>
    results = [utils.format_resource_search(result) for result in search_result['hits']]
  File "./app/utils.py", line 70, in format_resource_search
    'free': hit['free'],
KeyError: 'free'
```

I'm not sure exactly what to do to fix it... I assume reindex everything in algolia, but I'm not sure which commands to run to get there

Reply:
I am not sure as well about this as I have no experience in it but I think this could be the solution/related? https://www.algolia.com/doc/api-reference/api-methods/move-index/?language=python